lib/Controller/PageController.php

```diff
@@ -151,6 +151,22 @@ public function formPostEmb() {
 
         $tr = $this->showFormPost($userId, $pageId, true);
         $this->setEmbCsp($tr, $userId);
+
+        $hasActiveValidSession = false;
+        
+        if($this->userSession->getUser()){
+            $hasActiveValidSession = $this->userSession->getUser()->getUid() == $userId;
```

Comment:
Wouldn't this count a user who is using the external widget but is logged in as an internalWidgetBooking?

Reply:
Yes, good catch. I've changed the logic to use js to better detect it

Reply:
I think we need to double-check what the intention is behind the metric and what is considered internal. With the new implementation, if you access public booking form link and book a session it's logged as an internal booking, but I'm unsure whether that should count or it's only booking made from the dashboard that are considered internal.

Reply:
Just checking the original description it does say bookings made on the dashboard as internal, so the current implementation needs to change.

Reply:
> I think we need to double-check what the intention is behind the metric and what is considered internal. With the new implementation, if you access public booking form link and book a session it's logged as an internal booking, but I'm unsure whether that should count or it's only booking made from the dashboard that are considered internal.

The logging only happens in the controller function that handles the post request from the embed forms so the public page wouldn't be logged.

Reply:
Great!